packages/pluggable-widgets-tools/test-config/jest.enzyme-free.config.js

```diff
@@ -0,0 +1,32 @@
+const { join } = require("path");
+
+const projectDir = process.cwd();
+
+module.exports = {
+    clearMocks: true,
+    testRunner: "jest-jasmine2",
+    rootDir: join(projectDir, "src"),
+    setupFilesAfterEnv: [join(__dirname, "test-index-no-enzyme.js")],
+    testMatch: ["<rootDir>/**/*.spec.{js,jsx,ts,tsx}"],
```

Comment:
We might need to add this new file to the root `jest.config.js`.

Reply:
Can you explain a bit? I think I'm missing the context.

Reply:
Okay, some things now clear. In short - we don't need to move it to root. This file is for manual use. It needed only for those who don't need `enzyme` and removing `enzyme` form root config will be a breaking change.

Reply:
👍